Wikipedia/assets/pcs-html-converter/siteCSS.css

```diff
@@ -0,0 +1 @@
ll-caps}.navbox .navbar,.infobox .navbar{font-size:100%}.navbox .navbar{display:block}.navbox-title .navbar{float:left;text-align:left;margin-right:0.5em;width:6em}.plainlist ul{list-style:none;padding-left:0}.visualhide{position:absolute;left:-10000px;top:auto;width:1px;height:1px;overflow:hidden}.mw-parser-output .nowrap,.nowraplinks a,.nowraplinks .selflink,sup.reference a{white-space:nowrap}.mw-parser-output .infobox .nowrap{white-space:normal !important}.wrap,.wraplinks a{white-space:normal}.times-serif,span.texhtml{font-family:serif}span.texhtml{white-space:nowrap}.reflist ol.references{list-style-type:inherit}.refbegin-hanging-indents > ul,.refbegin-hanging-indents > dl{list-style-type:none;margin-left:0}.refbegin-hanging-indents > ul > li,.refbegin-hanging-indents > dl > dd{margin-left:0;padding-left:1.0em;text-indent:-1.0em;list-style:none}.flagicon img{min-width:25px}.cnotice{margin-bottom:0 !important}.mbox-text{padding:0.25em 0.9em}#content .hlist-separated li:after{font-size:0.8em;color:#333}.mw-parser-output .hlist ul,.mw-parser-output .hlist ol{padding-left:0}.mw-parser-output .hlist dd,.mw-parser-output .hlist dt{margin-right:0}.mw-parser-output .hlist dd:after{content:" · ";font-weight:bold}.mw-parser-output .hlist dt:after{content:":"}.heading-holder hlist li:after,.mw-parser-output .hlist dd:last-child:after,.mw-parser-output .hlist dt:last-child:after,.mw-parser-output .hlist li:last-child:after{content:none}.mw-parser-output .hlist dd dd:first-child:before,.mw-parser-output .hlist dd dt:first-child:before,.mw-parser-output .hlist dd li:first-child:before,.mw-parser-output .hlist dt dd:first-child:before,.mw-parser-output .hlist dt dt:first-child:before,.mw-parser-output .hlist dt li:first-child:before,.mw-parser-output .hlist li dd:first-child:before,.mw-parser-output .hlist li dt:first-child:before,.mw-parser-output .hlist li li:first-child:before{content:" (";font-weight:normal}.mw-parser-output .hlist dd dd:last-child:after,.mw-parser-output .hlist dd dt:last-child:after,.mw-parser-output .hlist dd li:last-child:after,.mw-parser-output .hlist dt dd:last-child:after,.mw-parser-output .hlist dt dt:last-child:after,.mw-parser-output .hlist dt li:last-child:after,.mw-parser-output .hlist li dd:last-child:after,.mw-parser-output .hlist li dt:last-child:after,.mw-parser-output .hlist li li:last-child:after{content:") ";font-weight:normal}.mw-parser-output .hlist ol{counter-reset:listitem}.mw-parser-output .hlist ol > li{counter-increment:listitem}.mw-parser-output .hlist ol > li:before{content:" " counter(listitem) " ";white-space:nowrap}.mw-parser-output .hlist dd ol > li:first-child:before,.mw-parser-output .hlist dt ol > li:first-child:before,.mw-parser-output .hlist li ol > li:first-child:before{content:" (" counter(listitem) " "}@media print{.navbox,.hatnote,.ambox{display:none}}.navigation-full-screen #mw-mf-page-left{width:100% !important;right:0 !important;opacity:1;display:block !important;visibility:visible !important}
```

Comment:
This varies by site so we'd have to bundle one for every language. Same comment as above though, depending on how the resulting migration looks it might be OK without it